week5/5.3/whisper.py

```diff
@@ -0,0 +1,35 @@
+"""
+whisper.
+By: Eliad Karni.
+The feature decode messages which contained in the file "logo.jpg".
```

Comment:
The term feature is incorrect here - write This code instead.  Feature usually refers to a piece of code inside a bigger code, not a standalone script